### PR TITLE
Remove multiline comment markers in single line comments

### PR DESCRIPTION
**Pull request by rscircus**

### CODE REVIEW

File: include/network.hpp [hunk 1]
```diff
@@ -11,33 +11,33 @@
...
-  int startlayerID; /* ID of the first layer on that processor */
-  int endlayerID;   /* ID of the last layer on that processor */
+  int startlayerID; // ID of the first layer on that processor 
+  int endlayerID;   // ID of the last layer on that processor 
 
-  int ndesign_global;   /* Global number of design vars  */
-  int ndesign_local;    /* Number of design vars of this local network block  */
+  int ndesign_global;   // Global number of design vars  
+  int ndesign_local;    // Number of design vars of this local network block  
   int ndesign_layermax; /* Max. number of design variables of all hidden layers
```
Comment (steffi7574) [inline include/network.hpp:28]: In my opinion, these comments should be either all /* */ or all //, no matter if single or multiple line. Don't like the switch.

File: src/config.cpp [hunk 2]
```diff
@@ -5,9 +5,9 @@
 #include <cstring>
 
 Config::Config() {
-  /* --- Set DEFAULT parameters of the config file options --- */
+  // Set DEFAULT parameters of the config file options ---
```
Comment (steffi7574) [inline src/config.cpp:8]: I have been using /* --- ... --- */ as a 'headline', to  visually separate the code. I'd appreciate if that stays so I find my stuff.

**Review by steffi7574 — COMMENTED** [review]
I am not a fan of this.. I'd prefer to leave the comments in /*  ... */ and headline style in /* --- ... ---*/, no matter if single or multiple line comment. I was using // mostly for additional comments that explain weird stuff behind a single code lines.

**Comment by rscircus** [comment]
As it just is a comment, I don't care. Please delete this merge request.

**Pull request closed without merging**